stable/insights-agent/templates/trivy/cronjob.yaml

```diff
@@ -58,6 +58,10 @@ spec:
             - name: TRIVY_INSECURE
               value: "true"
             {{ end }}
+            {{- if (index .Values "trivy-server" "enabled") }}
+            - name: TRIVY_SERVER_URL
```

Comment:
I think it would more flexible if we have
```
trivy:
  useServer: false
  serverUrl: https://trivy-server.insights-agent.svc.cluster.local:443
```
And here
```
            {{- if (index .Values "useServer" "enabled") }}
            - name: TRIVY_SERVER_URL
              value: {{ .trivy.serverUrl }}
            {{- end }}
```
The way it is set up, we cannot point an external trivy server, if the user wants to manage on its own

Reply:
done